chicago/views.py

```diff
@@ -333,35 +334,7 @@ def get_context_data(self, **kwargs):
             .order_by("-last_action")[:10]
         )
 
-        attendance = []
-        events = []
-
-        current_legislative_session = LegislativeSession.objects.get(
-            start_date__lt=timezone.now(), end_date__gt=timezone.now()
-        )
-
-        # fetch all events for the current legislative session for committees they're on
-        for membership in person.current_memberships.all():
-            events.extend(
-                ChicagoEvent.objects.filter(
-                    participants__organization=membership.organization,
-                    start_date__gte=membership.start_date,
-                )
-                .filter(start_date__gte=current_legislative_session.start_date)
-                .filter(participants__entity_type="person")
-                .distinct()
-                .prefetch_related("participants")
-            )
-
-        for e in sorted(events, key=attrgetter("start_time"), reverse=True):
-            attended = e.participants.filter(person_id=person.id).exists()
-            attendance.append(
-                {
-                    "event": e,
-                    "attended": attended,
-                }
-            )
-
+        attendance = person.full_attendance
         context["committee_count"] = len(person.current_memberships) - 1
         context["attendance"] = attendance
```

Comment:
consider if we need this at all or whether we can do person.full_attendance in the template?

Reply:
we need this to do some calculations on the number of events attended vs missed